src/App.tsx

```diff
@@ -27,6 +28,22 @@ const HomeStack = () => {
       <Stack.Screen name="WelcomeScreen" component={WelcomeScreen} />
       <Stack.Screen name="LoginScreen" component={LoginScreen} />
       <Stack.Screen name="SignUpScreen" component={SignUpScreen} />
+      <Stack.Screen
+        name="PasswordChanged"
+        component={PasswordChanged}
+        options={({ navigation }) => ({
+          headerTitle: '',
+          headerShadowVisible: false,
+          headerLeft: () => (
+            <TouchableOpacity onPress={() => navigation.goBack()} style={{ marginLeft: 15 }}>
+              <Image
+                source={require('../assets/leftArrow.png')}
```

Comment:
I am not sure, but there might be a way to improve these relative paths 🤔 can you look into it?

Reply:
yes, I'll try importing the image instead